allensdk/brain_observatory/ecephys/copy_utility/_schemas.py

```diff
@@ -18,10 +17,12 @@ class FileExists(InputFile):
 
 
 class FileToCopy(DefaultSchema):
-    source = InputFile(required=True, description='copy from here')
-    destination = OutputFile(required=True,
-                             description='copy to here (full path, not just '
-                                         'directory!)')
+    source = InputFile(
+            required=True,
+            description='copy from here')
+    destination = String(
```

Comment:
This seems like it should still be `OutputFile` and what actually needs to be fixed is something in?

https://github.com/AllenInstitute/AllenSDK/blob/5be964bf10c501c9066d8100bd2cbf8bcf8c0749/allensdk/brain_observatory/ecephys/copy_utility/__main__.py#L40